docs/manual-instrumentation.md

```diff
@@ -0,0 +1,43 @@
+# Manually instrument a .NET application
+
+The auto-instrumentation provides a base you can build on by adding your own
+custom instrumentation. By using both instrumentation approaches,
+you'll be able to present a more detailed representation of the logic
+and functionality of your application, clients, and framework.
+
+## Instrument using System.Diagnostics API
+
+For the list of steps required, see [link](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/docs/manual-instrumentation.md).
+
+## Instrument using OpenTracing API
+
+1. Add the `OpenTelemetry.Shims.OpenTracing` dependency to your project:
+
+    ```xml
+    <PackageReference Include="OpenTelemetry.Shims.OpenTracing" Version="1.0.0-rc9.10" />
+    ```
```

Comment:
Is it not enough to reference `OpenTracing`?

Reply:
So we need also to reference `System.Diagnostics.DiagnosticSource` `7.0.0`.

Reply:
changed in https://github.com/signalfx/splunk-otel-dotnet/pull/126/commits/8271315acf4cab6bd19f7fa0328e5e338bc7c28c
In version 0.5.0 of automatic instrumentation, no additional span is created (tags are added to existing span).
This seems to be fixed in latest versions of automatic instrumentation.